doc/source/whatsnew/v2.1.0.rst

```diff
@@ -749,6 +749,7 @@ I/O
 - Bug in :func:`json_normalize`, fix json_normalize cannot parse metadata fields list type (:issue:`37782`)
 - Bug in :func:`read_csv` where it would error when ``parse_dates`` was set to a list or dictionary with ``engine="pyarrow"`` (:issue:`47961`)
 - Bug in :func:`read_csv`, with ``engine="pyarrow"`` erroring when specifying a ``dtype`` with ``index_col`` (:issue:`53229`)
+- Bug in :func:`read_excel`, with ``engine="xlrd"`` (``xls`` files) erroring when file contains NaNs/Infs (:issue:`54564`)
```

Comment:
@mroeschke - should this be going into 2.2 at this point?

Reply:
Yeah probably. (Maybe "higher impact" bugs can go in 2.1 still)